tests/plugins/test_koji_import.py

```diff
@@ -130,6 +131,9 @@ def CGImport(self, metadata, server_dir, token=None):
         self.server_dir = server_dir
         return {"id": "123"}
 
+    def CGRefundBuild(self, cg, id, token):
```

Comment:
pylint is complaining about using `id` as a variable name here, since it's a Python builtin.